src/main/scala/com/fulcrumgenomics/umi/VanillaUmiConsensusCaller.scala

```diff
@@ -239,7 +239,7 @@ class VanillaUmiConsensusCaller(override val readNamePrefix: String,
   }
 
   /** Creates a consensus read from the given records.  If no consensus read was created, None is returned. */
-  protected[umi] def consensusFromSamRecords(records: Seq[SamRecord]): Option[VanillaConsensusRead] = {
+  protected[umi] def consensusFromSamRecords(records: Seq[SamRecord]): Option[(VanillaConsensusRead, Seq[SourceRead])] = {
```

Comment:
We now return the _filtered_ (by common alignment) source reads, from which we call the consensus UMI (to put into the RX tag).